flavors/template-Exasol-all-python-3.10/flavor_base/build_deps/packages/apt_get_packages

```diff
@@ -0,0 +1,9 @@
+coreutils|8.28-1ubuntu1
+locales|2.27-3ubuntu1.6
+tar|1.29b-2ubuntu0.4
+curl|7.68.0-1ubuntu2.19
+openjdk-11-jdk|11.0.19+7~us1-0ubuntu1~20.04.1
```

Comment:
22.04?

Reply:
Ah, I know what happens, the versions are actually ignored in the Dockerfile, this is a bug, but I will fix this separately.